### PR TITLE
Support course-specific overrides of certificate values

### PR DESCRIPTION
**Pull request by bryanlandia**
Allows Studio users to set values in Certificate HTML View Overrides
Advanced Settings field.

### CODE REVIEW

**Comment by bryanlandia** [comment]
I tried to rebase and force-push but not able to since the branch is protected.

**Review by grozdanowski — APPROVED** [review]
Looks good to me.

**Comment by bryanlandia** [comment]
thanks for the review @grozdanowski .  Could you see if you can give me admin rights on this repo or unlock this branch?  I want to rebase and force-push and GitHub's not letting me.

File: lms/templates/theme-variables.html [hunk 1]
```diff
@@ -357,7 +357,7 @@
     'cert_logo': get_value('certificates', {}).get('cert_logo', get_value('logo_positive', static('images/branding/brand-logo.svg'))),
     'logo_width': get_value('certificates', {}).get('cert_logo_width', '160px'),
     'platform_name': translate(get_value('certificates', {}).get('platform_name', get_value('PLATFORM_NAME', '')), default=_('Platform Name')),
-    'certificate_banner_text': get_value('certificates', {}).get('certificate_banner_text', course_context.get('document_banner', '')),
+    'certificate_banner_text': translate(get_value('certificates', {}).get('certificate_banner_text', course_context.get('document_banner', ''))),
```
Comment (OmarIthawi) [inline lms/templates/theme-variables.html:360]: what's the purpose of `translate` here? I don't think it's relevant. right?

**Review by OmarIthawi — APPROVED** [review]
LGTM.

File: lms/templates/theme-variables.html (hunk 1 above)
Comment (bryanlandia) [reply lms/templates/theme-variables.html:360]: @OmarIthawi the `translate` is coming from a commit I still need to push from a rebase.  All the other values are `translate`d, so I was following the pattern there.  It probably wouldn't be relevant unless the value is not set and we fall back to the default, in which case we may have a translation string in place.

I don't have permission to force-push to this repo though.

Comment (bryanlandia) [reply lms/templates/theme-variables.html:360]: Hmm, wait,  I was confused. I forgot, my rebase was only for squashing. 😆 Still, point stands about the translate.
I'll merge as is tomorrow unless @OmarIthawi or @grozdanowski one of you might be able to give me force-push perms on this repo.

**Comment by OmarIthawi** [comment]
@bryanlandia the `hawthorn/tahoe-course-cert-overrides` branch is protected because it matches the `hawthorn/*` pattern.

The intention was to protect `hawthorn/tahoe` and `hawthorn/nyif` but certainly not your branch. I'm afraid it's not worth removing the rule so we have three options:

 - Merge without squashing.
 - Open a new branch with a different prefix e.g. `bryan/tahoe-course-cert-overrides`
 - Use the GitHub native "Squash and merge" feature:
   <kbd>image</kbd>

**Pull request merged**